### DISCUSSION TITLE
posts/031-weekly-report-dec-2/

### DISCUSSION THREAD

**Discussion by giscus** [Yamabuki-bakery/tasty-discussion#18] [Announcements]
# posts/031-weekly-report-dec-2/

🏆本週最香榨菜：有人餵你吃愛因斯坦引力場方程

https://example.com

**Comment by fmgjz** [comment]
**坐等闭源lib（**

**Reply by PermeationLoop** [reply]
正在學了，別急

**Comment by PermeationLoop** [comment]
喜報：sora 用 canvas 撕出來一坨大的，然後半途而廢了．
俺難得的 idea 就這麼暴死了😭

**Comment by PermeationLoop** [comment]
喜報，本羣的藝術家們聯合起來，用 Three.js 撕出來一坨棒的，感謝 [🥰LuxAeris](https://github.com/NagatoCute)！

<img width="1933" height="1223" alt="3D 場景和貼圖" src="https://gist.github.com/user-attachments/assets/209c5eb6-d623-4a61-a158-365e6d61678f" />

掀起技術美術和 web 開發的融合革命，指日可待！
